### ISSUE TITLE
"Require" quick fix for keywords namespaced with an alias

### ISSUE THREAD

**Issue by spieden** [cursive-ide/cursive#1552]
Currently if I type a namespace aliased symbol like `strs/join`, Cursive's inspection highlights it and offers a quick fix to require `clojure.string` for me. However if I type `::my-specs/thing` I get no such help and am stuck manually writing a require.

Given that `::my-specs/thing` is not syntactically valid ("Invalid token") without a `my-specs` namespace alias, it seems like it should be akin to the symbol case.

**Comment by metametadata** [comment]
I use specs extensively and this is a very frustrating omission.

**Comment by cmf** [comment]
Fix released in 2026.1-eap1 on 2026-02-01.

**Comment by cmf** [comment]
Thanks for the feedback, I've filed #3080 and #3081 for those.

**Comment by metametadata** [comment]
Thank you. 

The problem I see now is that it suggests several namespaces to me, even though only one of them contains the needed spec.

**Comment by cmf** [comment]
@metametadata Can you give me an example of what you mean here? Do you mean that the fix suggests namespaces that use that alias in other places, but do not contain a definition of the spec?

**Comment by metametadata** [comment]
> Do you mean that the fix suggests namespaces that use that alias in other places, but do not contain a definition of the spec?

Exactly. For example, here only specs1 defines the spec, but specs2 is also suggested unexpectedly:

<img width="424" height="166" alt="Image" src="https://github.com/user-attachments/assets/4602fc62-a642-4a6e-a7f1-23ad4493d647" />

**Comment by cmf** [comment]
The problem is that this feature doesn't just work for keywords that define things, like specs. It also works for e.g. namespaced keywords that you'd use as map keys. I guess it should check that `::specs2/version exists` at least, but I don't think I can limit it to just where something is defined in that namespace. In this case, do you have both `:app.specs1/version` and `:app.specs2/version` somewhere, even if only one of them is used to define a spec?

**Comment by metametadata** [comment]
> In this case, do you have both `:app.specs1/version` and `:app.specs2/version` somewhere, even if only one of them is used to define a spec?

Unlikely, because I rarely use namespaced keywords for non-specs and the majority of specs are defined in the dedicated `*.specs` namespaces.

> I guess it should check that ::specs2/version exists at least

I guess it would resolve such problems for me.
